torchvision/prototype/transforms/_misc.py

```diff
@@ -140,14 +140,25 @@ def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
         return F.gaussian_blur(inpt, self.kernel_size, **params)
 
 
-# TODO: Enhance as described at https://github.com/pytorch/vision/issues/6697
-class ToDtype(Lambda):
-    def __init__(self, dtype: torch.dtype, *types: Type) -> None:
+class ToDtype(Transform):
+    def __init__(self, dtype: Union[torch.dtype, OrderedDictType[Type, torch.dtype]]) -> None:
+        super().__init__()
+        if isinstance(dtype, torch.dtype):
+            dtype = OrderedDict([(torch.Tensor, dtype)])
```

Comment:
`fill` uses default dicts here. Should we align?

Reply:
No. `fill` has a default value whereas `dtype` has none.